compiler/rustc_middle/src/ty/cast.rs

```diff
@@ -3,13 +3,12 @@
 
 use crate::ty::{self, Ty};
 
-use rustc_ast as ast;
 use rustc_macros::HashStable;
 
 /// Types that are represented as ints.
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum IntTy {
-    U(ast::UintTy),
+    U(ty::UintTy),
```

Comment:
I did not see a re-export of `rustc_type_ir::UintTy` in `rustc_ast`. Are both the same type?

Reply:
I guess it could make sense to do that, but that means having `rustc_ast` depend on `rustc_type_ir`... 🤷‍♂️